public/js/app.js

```diff
@@ -815,14 +822,14 @@
         $messagebox.val('');
 
         if (value) {
-          if (pvtmsgto) {
```

Comment:
I didn't want to parse slash commands in private message, but i can live with it.